design-tokens/Decathlon-Origins-Light.json

```diff
@@ -928,5 +928,43 @@
       "value": "{TextCase.Uppercase}",
       "type": "textCase"
     }
+  },
+  "BorderRadius": {
+    "xs": {
+      "value": "{BorderRadius.0}",
+      "type": "borderRadius"
+    },
```

Comment:
Does it mean that when you set the size of your border-radius to xs using the `borderRadius-xs` token, actually it doesn't set any border-radius? It seems a bit counter-intuitive to me 😬

Reply:
+1